### ISSUE TITLE
BalenaEtcher and Emteria Installer fail because of permission problems

### ISSUE THREAD

**Issue by Tercus** [TheAssassin/AppImageLauncher#297]
**Bug description**
When running BalenaEtcher the software asks for root permission to format the selected drive. After entering the password the operation still fails. The console (it is an electron app) shows following error:
```
/tmp/.mount_balena9kYAkM/resources/app.asar/node_modules/raven-js/dist/raven.js:58 {"stack":"Error: The elevated process died unexpectedly\n    at createError (file:///tmp/.mount_balena9kYAkM/resources/app.asar/generated/gui.js:1091:19)\n    at Object.createUserError (file:///tmp/.mount_balena9kYAkM/resources/app.asar/generated/gui.js:1112:12)\n    at file:///tmp/.mount_balena9kYAkM/resources/app.asar/generated/gui.js:5973:30","message":"The elevated process died unexpectedly","description":"The process error code was 126","report":false,"image":"lineage-16.0-20191222-UNOFFICIAL-KonstaKANG-rpi3.img"}
```
The Emteria installer needs to be run with sudo, but this fails. Probably related to the already reported bug #266.

**Expected behavior**
Run BalenaEtcher without sudo and get asked for permission. Enter permission and be able to flash a USB-drive.
Run Emteria installer with sudo and be able to flash a USB-drive

**To Reproduce**
For BalenaEtcher:
- download BalenaEtcher
- Try to flash a USB-drive
- get asked for password
- get an error message

For Emtera installer:
- download Emtera installer
- try to run it with sudo
- get error message

**System, software and AppImage information**
```
System:    Host: tobias-X570 Kernel: 5.5.0-050500rc5-generic x86_64 bits: 64 Desktop: KDE Plasma 5.17.5
           Distro: KDE neon User Edition 5.17
Machine:   Device: desktop System: Gigabyte product: X570 AORUS MASTER v: -CF serial: N/A
           Mobo: Gigabyte model: X570 AORUS MASTER v: x.x serial: N/A
           UEFI: American Megatrends v: F11 date: 12/06/2019
CPU:       8 core AMD Ryzen 7 3800X (-MT-MCP-) speed/max: 1888/3900 MHz
Graphics:  Card: Advanced Micro Devices [AMD/ATI] Device 7340
           Display Server: x11 (X.Org 1.19.6 )
           drivers: ati,amdgpu (unloaded: modesetting,fbdev,vesa,radeon)
           Resolution: 1920x1080@71.91hz
           OpenGL: renderer: AMD NAVI14 (DRM 3.36.0, 5.5.0-050500rc5-generic, LLVM 9.0.1)
           version: 4.6 Mesa 20.0.0-devel (git-6bafd23 2020-01-12 bionic-oibaf-ppa)
Network:   Card-1: Intel Device 2723 driver: iwlwifi
           Card-2: Intel I211 Gigabit Network Connection driver: igb
           Card-3: Realtek Device 8125 driver: r8169
Drives:    HDD Total Size: 500.1GB (85.3% used)
Info:      Processes: 368 Uptime: 12:14 Memory: 4549.5/32126.5MB Client: Shell (bash) inxi: 2.3.56
```
**AppImageLauncher version used:**
1.2.2+dfsg-1+18.04+bionic+build4 amd64
got it from default KDE neon repository (should be ubuntu)

**List of AppImages tried:**
- Balena Etcher in many different versions
- Emtera installer
- KDEnlive (tried running it with sudo)

**Comment by TheAssassin** [comment]
As you say, it's related to #266. We have to fix #266 and then can continue here.

In the meantime, you can try to update AppImageLauncher to the latest versions, there's been a lot of fixes in the meantime, 1.2.2 is really old.

**Comment by TheAssassin** [comment]
Closing as obsolete. See #361 for more information.
